alphastats/gui/pages/02_Import Data.py

```diff
@@ -1,76 +1,141 @@
 import streamlit as st
-import os
-import io
-
-try:
-    from alphastats.gui.utils.import_helper import (
-        import_data,
-        save_plot_sampledistribution_rawdata,
-        display_loaded_dataset,
-        load_sample_data,
-        empty_session_state,
-    )
-    from alphastats.gui.utils.ui_helper import sidebar_info
-
-except ModuleNotFoundError:
-    from utils.ui_helper import sidebar_info
-    from utils.import_helper import (
-        import_data,
-        save_plot_sampledistribution_rawdata,
-        display_loaded_dataset,
-        load_sample_data,
-        empty_session_state,
-    )
 
-from streamlit.runtime import get_instance
-from streamlit.runtime.scriptrunner.script_run_context import get_script_run_ctx
+from alphastats import DataSet
+from alphastats.gui.utils.options import SOFTWARE_OPTIONS
+
+from alphastats.gui.utils.import_helper import (
+    save_plot_sampledistribution_rawdata,
+    load_example_data,
+    empty_session_state,
+    load_softwarefile_df,
+    show_metadata_file_uploader,
+    show_loader_columns_selection,
+    load_proteomics_data,
+    load_options,
+    show_select_sample_column_for_metadata,
+    init_session_state,
+)
+from alphastats.gui.utils.ui_helper import sidebar_info
+
+init_session_state()
 
-runtime = get_instance()
-session_id = get_script_run_ctx().session_id
-# session_info = runtime._session_mgr.get_session_info(session_id)
-# TODO: remove this line at some point if we really don't need it
+sidebar_info()
 
-user_session_id = session_id
-st.session_state["user_session_id"] = user_session_id
+st.markdown("### Start a new session")
+st.write(
+    "Start a new session will discard the current one (including all analysis!) and enable importing a new dataset."
+)
+st.write("To explore AlphaPeptStats you may also load an example dataset.")
 
-if "loader" not in st.session_state:
-    st.session_state["loader"] = None
+c1, c2 = st.columns(2)
```

Comment:
I like the new session or new session with example dataset.
Longterm I would however prefer that the session doesn't need to be reset 'manually', i.e. just have the interface for loading data and button to load the sample data, with no need to reset the session (this should just happen automatically)